packages/wrangler/src/secret/index.ts

```diff
@@ -57,7 +57,15 @@ async function createDraftWorker({
 	accountId: string;
 	scriptName: string;
 }) {
-	// TODO: log a warning
+	const confirmation = await confirm(
+		`No Worker called "${scriptName}" appears to exist. Do you want to create a new Worker and add secrets to it?`,
```

Comment:
are we sure this is the behaviour we want here? Should we instead users to chose a name for this new Worker? Prompting that "non-existent-worker" does not exist feels a bit shady to me?

Reply:
What do you mean by shady haha? Before the behaviour was to silently create a new worker with whatever name you passed in - if you typo'd the name, you'd end up with a new worker with the typo name. But also some people actually did want to create new workers + upload secrets at them same time, which is why this behaviour was added. 

I might tweak the wording tho:

`There doesn't seem to be a worker called "${scriptName}". Do you want to create a new Worker with that name and add secrets to it?`

Reply:
for postery, as per our async convo, turns I missunderstood how this works, and what you have in place here is totes 👌